api/envoy/config/response_map/v3/response_map.proto

```diff
@@ -0,0 +1,89 @@
+syntax = "proto3";
+
+package envoy.config.response_map.v3;
+
+import "envoy/config/accesslog/v3/accesslog.proto";
+import "envoy/config/core/v3/base.proto";
+import "envoy/config/core/v3/substitution_format_string.proto";
+
+import "google/protobuf/struct.proto";
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.response_map.v3";
+option java_outer_classname = "ResponseMapProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: ResponseMap]
+// Response map filter :ref:`configuration overview <config_response_map>`.
+
+// The configuration to filter and change HTTP responses.
+// [#next-free-field: 6]
+message ResponseMapper {
```

Comment:
This isn't different from [ResponseMapper](https://github.com/envoyproxy/envoy/blob/master/api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto#L620) in HCM, why not just use it?

Reply:
I captured my thoughts here https://docs.google.com/document/d/1THyRnepx89FbK2VVaqoEHXv2knbm_lsLx2XgDZ7FxDw and added a link to the PR description

Reply:
Thanks, can you give us comment access to the doc?

Reply:
Done